sui_programmability/framework/sources/Governance/Genesis.move

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module Sui::Genesis {
+    use Std::Vector;
+
+    use Sui::Coin;
+    use Sui::SUI;
+    use Sui::SuiSystem;
+    use Sui::TxContext::TxContext;
+    use Sui::Validator;
+
+    /// The initial amount of SUI locked in the storage fund.
+    /// 10^14, an arbitrary number.
+    const INIT_STORAGE_FUND: u64 = 100000000000000;
+
+    /// Initial value of the lower-bound on the amount of stake required to become a validator.
+    const INIT_MIN_VALIDATOR_STAKE: u64 = 100000000000000;
+
+    /// Initial value of the upper-bound on the amount of stake allowed to become a validator.
+    const INIT_MAX_VALIDATOR_STAKE: u64 = 100000000000000000;
+
+    /// Initial value of the upper-bound on the number of validators.
+    const INIT_MAX_VALIDATOR_COUNT: u64 = 100;
+
+    /// Basic information of Validator1, as an example, all dummy values.
+    const VALIDATOR1_SUI_ADDRESS: address = @0x1234;
+    const VALIDATOR1_NAME: vector<u8> = b"Validator1";
+    const VALIDATOR1_IP_ADDRESS: vector<u8> = x"00FF00FF";
```

Comment:
Could we have URLs for this? (domain, port at least).

Reply:
I leave it to the validator to decide how to interpret it (with `vector<u8>`). 
Or are you suggesting we should settle on what's included in the ip address field?

Reply:
Lets call it NET_ADDRESS and leave it open to interpretation for the moment.